audit.sql

```diff
@@ -131,18 +131,18 @@ BEGIN
     IF (TG_OP = 'UPDATE' AND TG_LEVEL = 'ROW') THEN
         old_r = to_jsonb(OLD);
         new_r = to_jsonb(NEW);
-        audit_row.row_data = old_r - excluded_cols;
+        audit_row.row_data = old_r #- excluded_cols;
```

Comment:
Looks like the `-` operator which takes a list of keys as an argument was only introduced in Postgres 10. Note that this is not the same as `#-` operator. From the docs:

> `- text[]` 

Delete multiple key/value pairs or string elements from left operand. Key/value pairs are matched based on their key value. 

> `#- text[]` 

Delete the field or element with specified path (for JSON arrays, negative integers count from the end)

Reply:
So, how to resolve https://github.com/hasura/graphql-engine/issues/2732 ?
I'm running Postgres 9.6 and applied this audit caused the error as I posted in that issue.

Reply:
@hynra we will need to write something for `9.6` which works like `-` and use that conditionally. The other option is to clone `audit.sql` and remove the `ignored_columns` part of the codebase (this is what requires `-` operator to be present).